sql/attrib_entries.sql

```diff
@@ -330,6 +330,22 @@ INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (541, '_stop_codon_rt', 'Stop Codon Readthrough', '');
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (542, 'citation_source', 'Citation source', 'Variant citation data source');
 INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (544, 'phenotype_type', 'Phenotype type', 'Type of the phenotype information');
+INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (545, 'external_plant_links', 'external_plant_links', 'Links to external plant related sources');
+INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (546, 'coord_system_tag', 'coord_system_tag', 'Coord_system_tag -- a way to store information about  seq_region\'s virtual coord_system names for single level assemblies.
+I.e., we have an assembly with toplevel seq_regions ("primary_assembly") only,
+and we want to store the initial type of  seq_regions like contig, scaffold, supercontig, chromosome, etc.');
+INSERT IGNORE INTO attrib_type (attrib_type_id, code, name, description) VALUES  (547, 'sequence_location', 'sequence_location', 'To identify  sequence locations / cellular compartments  DNA sequence comes from.
```

Comment:
The descriptions for these 2 inserts do not match the master database. There are additional carriage returns in the master.